bin/rename_fasta.py

```diff
@@ -126,6 +128,18 @@ def main():
     rename_parser.set_defaults(func=rename)
 
     restore_parser = subparser.add_parser("restore")
+    restore_parser.add_argument(
+        "--from-restore",
+        help="Name of column FROM what do renaming",
+        type=str,
+        choices=['temporary', 'original', 'short']
+    )
+    restore_parser.add_argument(
+        "--to-restore",
+        help="Name of column TO what do renaming",
+        type=str,
+        choices=['temporary', 'original', 'short']
+    )
```

Comment:
Could you add a comment line or include help message examples of the expected string for each name type?